benchmarks/benchmarks/merge_concat.py

```diff
@@ -45,13 +45,15 @@ class Concatenate:
     cube_list: CubeList
 
     def setup(self):
-        source_cube = realistic_4d_w_everything()
-        second_cube = source_cube.copy()
-        first_dim_coord = second_cube.coord(dimensions=0, dim_coords=True)
-        first_dim_coord.points = (
-            first_dim_coord.points + np.ptp(first_dim_coord.points) + 1
-        )
-        self.cube_list = CubeList([source_cube, second_cube])
+        source_cube = realistic_4d_w_everything(lazy=True)
+        self.cube_list = CubeList([source_cube])
+        for _ in range(24):
```

Comment:
The concatenate benchmarks now concatenate 25 cubes, to make the benchmark more realistic but not too time-consuming.

Reply:
I tried this branch with one year of high resolution data ( 12 cubes to concatenate) and I get a decrease in time of about 90 seconds with respect to main branch :+1:

Reply:
Thanks for testing @sloosvel! Do you know what the total time was as well?

Reply:
I get around one minute for the parallel concatenation and two minutes / two minutes and a half for the main branch